src/main/java/com/minecolonies/entity/ai/citizen/farmer/EntityAIWorkFarmer.java

```diff
@@ -138,7 +138,7 @@ private AIState prepareForFarming()
 
         if (currentField.needsWork())
         {
-            if (!checkForHoe() && canGoPlanting(currentField, building))
+            if(!checkForHoe() && canGoPlanting(currentField, building))
```

Comment:
That missing space is on me, habit of removing it when IDE's add it that I picked up in languages that would flip for having it.